styles.css

```diff
@@ -19,7 +23,7 @@ body {
 #intro {
   display: inline;
   top: 120px;
-  width: 327px;
+  width: 100%;
   height: 124px;
   bottom: 500px;
 }
```

Comment:
Great work @Solobarine 👏

- [ ] [Optional] There is overlapping text on your design and the `<h1>` and `<p>` is too close. Please try to make this design close to the one provided in Figma.
- Your design
![solo design](https://user-images.githubusercontent.com/63917725/177010903-c61ea0ba-ad13-45b3-ac77-599c3d35e543.PNG)